src/guides/v2.3/ui_comp_guide/components/ui-image-preview.md

```diff
@@ -41,8 +41,8 @@ The ImagePreview component has dependencies on the following components:
 
 Extends `Column`:
 
-*  [app\code\Magento\Ui\view\base\web\js\grid\columns\image-preview.js]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js)
-*  [app\code\Magento\Ui\view\base\web\templates\grid\columns\image-preview.html]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/templates/grid/columns/image-preview.html)
+*  [app\code\Magento\Ui\view\base\web\js\grid\columns\image-preview.js](https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js)
```

Comment:
it is not clear why the link must point to a `develop` branch

Reply:
Because they don't exist in mainline and you wanted to keep these 404s as indicators of topics that need review.